### ISSUE TITLE
loss的计算

### ISSUE THREAD

**Issue by yyHaker** [persistforever/cifar10-tensorflow#1]
![screenshot from 2018-04-26 16-03-16](https://user-images.githubusercontent.com/18585628/39293243-fbaecc68-496a-11e8-9a35-2d6bcc97be53.png)

![screenshot from 2018-04-26 15-58-05](https://user-images.githubusercontent.com/18585628/39293275-0c2cc874-496b-11e8-9be9-84b8f42c1af1.png)

既然喂给网络一个batch的数据，计算的不应该是一个batch的损失吗？为什么后面计算平均损失的时候需要再乘以batch_size?

**Comment by Site1997** [comment]
@yyHaker I agree with you，maybe he intended to use "tf.reduce_mean" instead of "tf.reduce_sum"
